GoogleForCreators/web-stories-wp
# Background Audio: Display captions in editor on play

## Context



## Summary

Reuse the `VideoCaptionsLayer` component to render captions for background audio.  

## Relevant Technical Choices

Change Video to media, as captions are used for audio and video captions. 

## To-do



## User-facing changes

https://user-images.githubusercontent.com/237508/152998936-292f40b5-4ef7-456c-9d14-6aca39b8a2c7.mov

## Testing Instructions






This PR can be tested by following these steps:

1. Create a story editor. 
2. Create a new page. 
3. Select background.
4. Add background audio. 
5. Add background audio caption. 
6. Play background audio. 
7. See captions. 

Please ensure that enhanced captions are enabled. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10439
